pkg/serviceprovider/lookup.go

```diff
@@ -84,21 +86,21 @@ func (l GenericLookup) Lookup(ctx context.Context, cl client.Client, matchable M
 		return result, fmt.Errorf("failed to list the potentially matching tokens: %w", err)
 	}
 
-	lg.Info("lookup", "potential_matches", len(potentialMatches.Items))
+	lg.V(logs.DebugLvl).Info("lookup", "potential_matches", len(potentialMatches.Items))
```

Comment:
I would much rather have a facade that I could use to say `logs.Debug(lg, msg, keysAndValues)` as opposed to the `lg.V(level).Info(...)` syntax, but I'm willing to surrender if you think otherwise :)

Reply:
I also like more the slf4j way than this. But I'm not sure it's worth it. It will bring another dependency and it will become less go-like.

Reply:
Let me surrender here for now then :) Let's see how `lg.V(logs.DebugLevel).Info(...)` works for us..